examples/tti/tti_operators.py

```diff
@@ -158,7 +158,7 @@ def ForwardOperator(model, u, v, src, rec, damp, data, time_order=2,
         kwargs.pop('dle', None)
 
         op = Operator(nt, m.shape, stencils=stencils, subs=[subs, subs],
-                      spc_border=spc_order/2, time_order=time_order,
+                      spc_border=spc_order, time_order=time_order,
```

Comment:
Non-symmetric stencil in TTI, I don't know when the /2 came back but it will segfault with it